client-src/elements/form-field-specs.js

```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
```

Comment:
Maybe:

```suggestion
    label: 'TAG Specification Review',
```

To distinguish this from the earlier TAG Early Design Review.

Reply:
Yes, I was wondering about that.  I don't think we have any dependency on the label otherwise.